### PR TITLE
Fix linting, edit Linting rules. 

### PR DESCRIPTION
**Pull request by JaiS99**
## Issue number

Relevant issue number : #415

## Please check the following

- [ ] Do the tests still pass? (see [Run the Tests](https://github.com/cuttle-cards/cuttle/blob/main/docs/setup-development.md#run-the-tests))
- [X] Is the code formatted properly? (see [Linting (Formatting)](https://github.com/cuttle-cards/cuttle/blob/main/docs/setup-development.md#linting-formatting))
- For New Features:
  - [ ] Have tests been added to cover any new features or fixes?
  - [ ] Has the documentation been updated accordingly?

## Please describe additional details for testing this change
With Reference to issue https://github.com/cuttle-cards/cuttle/issues/415,
Made a change for linting

### CODE REVIEW

**Comment by JaiS99** [comment]
Please have a look at .eslintrc.js
Code has been formatted

**Review by itsalaidbacklife — COMMENTED** [review]
@JaiS99 the addition of the html-closing-bracket-newline rule will require re-running `npm run lint:fix`. Lastly could you please bump the `version` field in `package.json` to 4.3.21?

**Comment by JaiS99** [comment]
Updated package.json and re-ran `npm run lint:fix`, spewed out a warning:  

```
/workspaces/cuttle/src/components/RulesDialog.vue
  135:1  warning  This line has a length of 203. Maximum allowed is 120  'max-len'
```

but re-ran successfully

File: .eslintrc.js [hunk 1]
```diff
@@ -39,9 +38,26 @@ module.exports = {
...
+      }
+    }],
+    'vue/first-attribute-linebreak': ['error', {
+      'singleline': 'beside',
+      'multiline': 'below'
+    }],
   },
   overrides: [
-    // Vue specific rules
+ // Vue specific rules
```
Comment (itsalaidbacklife) [inline .eslintrc.js:60]:
```suggestion
```

**Review by itsalaidbacklife — APPROVED** [review]
Nice work. Thank you for contributing to Cuttle!

File: .eslintrc.js [hunk 2]
```diff
@@ -39,9 +38,25 @@ module.exports = {
     'no-debugger': process.env.NODE_ENV === 'production' ? 'warn' : 'off',
     quotes: ['error', 'single', { avoidEscape: true, allowTemplateLiterals: true }],
     semi: ['error', 'always'],
+
```
Comment (itsalaidbacklife) [inline .eslintrc.js:41]:
```suggestion
```

**Pull request merged**